utils/problems.py

```diff
@@ -415,6 +451,34 @@ async def fetch_problems_solved_and_rank(
         submit_stats_global = matched_user["submitStatsGlobal"]
         ac_submission_num = submit_stats_global["acSubmissionNum"]
 
+        languages_problem_count = matched_user["languageProblemCount"]
+
+        tags_problem_count = matched_user["tagProblemCounts"]
+        advanced_tags_count = tags_problem_count["advanced"]
+        intermediate_tags_count = tags_problem_count["intermediate"]
+        fundamental_tags_count = tags_problem_count["fundamental"]
+
+        languages_problem_count_list = []
```

Comment:
There shouldn't really be a need to add the type name to the variable name (in this case `list`). Also, to maintain consistency with the naming of the graphql query, just call this variable `language_problem_count`. Similarly for the rest, e.g. your `advanced_tag_list` variable, rename to `tagProblemCountsAdvanced`